awx/ui_next/src/components/Lookup/Lookup.jsx

```diff
@@ -76,21 +80,38 @@ class Lookup extends React.Component {
     this.saveModal = this.saveModal.bind(this);
     this.getData = this.getData.bind(this);
     this.clearQSParams = this.clearQSParams.bind(this);
+    this.toggleDropdown = this.toggleDropdown.bind(this);
   }
 
   componentDidMount() {
-    this.getData();
+    const { onLoadCredentialTypes } = this.props;
+    if (onLoadCredentialTypes) {
```

Comment:
This prop isn't passed in from CredentialsLookup, so this `if` seems unnecessary